lib/components/article.js

```diff
@@ -4,6 +4,13 @@ import Embed from './embed';
 
 function renderItem (item) {
   if (item.type === 'paragraph') {
+    const hasContent = item.children.filter(item => item.type !== 'linebreak' &&
+        item.content && item.content.trim()).length > 0;
+
+    if (!hasContent) {
+      return '';
+    }
+
```

Comment:
@iefserge OOhhh! That's interesting - it's the `&nbsp;` getting transformed to the non-breaking space char (http://www.wikiwand.com/en/Non-breaking_space) rather than the normal space. So doing `require('./dist')()('<p>&nbsp; </p>')[0]` will return two spaces of different kind. Both spaces are being handles properly by `.trim()` so doesn't matter.